website/docs/docs/mesh/iceberg/snowflake-iceberg-support.md

```diff
@@ -7,10 +7,10 @@ description: Understand Snowflake support for Apache Iceberg.
 
 dbt supports materializing the table in Iceberg table format in two different ways:
 
-- the model configuration field table_format = 'iceberg' (legacy)
-- catalog integration in the model/resource/dbt_project.yml configuration
+- the model configuration field `table_format = 'iceberg'` (legacy)
+- catalog integration in the `model/resource/dbt_project.yml` configuration
```

Comment:
@b-per should this be "catalog integration in the property or project file configurations?" This seems like a directory but would the dbt_project.yml ever be in this directory?

Reply:
It's not a directory, it's presenting 3 different options